client/store/flash/mutations.js

```diff
@@ -0,0 +1,10 @@
+export const FLASH_MESSAGE = 'FLASH_MESSAGE';
+export const CLEAR_FLASH_MESSAGE = 'CLEAR_FLASH_MESSAGE';
+export default {
+  [FLASH_MESSAGE](state, message) {
+    state.messages = [...state.messages, message];
+  },
+  [CLEAR_FLASH_MESSAGE](state, id) {
+    state.messages = state.messages.filter((message) => message.id != id);
```

Comment:
Codacy found an issue: [Expected '!==' and instead saw '!='.](https://app.codacy.com/manual/sam-spain/full-stack-js/pullRequest?prid=6207934)